### ISSUE TITLE
Note submission feature

### ISSUE THREAD

**Issue by cyrus01337** [haotian2006/MyDocs#7]
Tracking issue.

Just an idea I had that's separate to dev tags that may prove useful in case others or I get lazy and don't particularly want to contribute via Git and Github.

There exists a webpage with form fields (limited by character limits) that when submitted, automatically creates a submission in the form of an issue (simple)/PR (potentially annoying).

**Comment by haotian2006** [comment]
For non github accounts we could use their discord ID instead or just asking for a name

**Comment by cyrus01337** [comment]
True, with that I'm sure we can procure other details like username, etc. though I've never had the "pleasure" of working with the Discord API directly. If it's also something you've not dabbled with then freezing that feature for now is plausible.

**Comment by haotian2006** [comment]
yeah it could work
